ceph-mon/lib/charms/ceph_mon/v0/ceph_cos_agent.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""
+Library for handling ceph observability integrations
+"""
+
+import logging
+import socket
+import tenacity
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import charm
+
+from charms.grafana_agent.v0 import cos_agent
+from charms_ceph import utils as ceph_utils
+
+# The unique Charmhub library identifier, never change it
+LIBID = "ac526775f8ed42ebb46b231dc00519d3"
+
+# Increment this major API version when introducing breaking changes
+LIBAPI = 0
+
+# Increment this PATCH version before using `charmcraft publish-lib` or reset
+# to 0 if you are raising the major API version
+LIBPATCH = 1
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_CEPH_JOB = {
+    "metrics_path": "/metrics",
+    "static_configs": [{"targets": ["*:9283"]}],
+}
+DEFAULT_CEPH_METRICS_ENDPOINT = {
+    "path": "/metrics",
+    "port": 9283,
+}
+DEFAULT_ALERT_RULES_RELATIVE_PATH = "files/prometheus_alert_rules"
```

Comment:
I believe these DEFAULT_* constants are only used for the legacy metrics integration and could be removed here

Reply:
ack, thanks for the catch. Let me fix this.